doc/source/kaev_niederlausitz.md

```diff
@@ -0,0 +1,55 @@
+# Kommunaler Abfallentsorgungsverband Niederlausitz
+
+Support for schedules provided by [kaev.de](https://www.kaev.de/).
+
+## Configuration via configuration.yaml
+
+```yaml
+waste_collection_schedule:
+  sources:
+    - name: bsr_de
+      args:
+        abf_strasse: STRASSE
+        abf_ort: ORT
+		abf_ot: ORTSTEIL/GEMEINDETEIL/ORT
+```
+
+### Configuration Variables
+
+**abf_strasse**<br>
+*(string) (required)*
+
+**abf_ort**<br>
+*(string) (required)*
+
+**abf_ot**<br>
+*(string) (required)*
+
+## Example
+
+```yaml
+waste_collection_schedule:
+  sources:
+    - name: kaev_niederlausitz
+      args:
+        abf_strasse: "Zieckau"
+        abf_ort: "Luckau"
+		abf_ot: "Zieckau"
+```
+
+```yaml
+waste_collection_schedule:
+  sources:
+    - name: kaev_niederlausitz
+      args:
+        abf_strasse: "Bersteweg"
+        abf_ort: "Luckau"
+		abf_ot: "Luckau"
+```
+
+
+## How to get the source arguments
+
+
+1. Go to your calendar at `https://www.kaev.de/Info-und-Service/Tourenplan/Tourenplan-Abfalltermine.html`.
+2. Enter your location.
```

Comment:
I followed the link and entered "Staakow" in the search field. The site returned results for that. But how do I get the values for the 3 fields from that?

Would it be easier to use just one configuration option for the source which is exactly the content of the search field?

Reply:
So mean remove abf_ort, abf_ot and abf_strasse and just an abf_search where the user has to specify the full string from search? Currently, it's more like you enter everything you know and then the script is trying to find the right option.